modules/neo4j/custom-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'java'
```

Comment:
I would suggest to use a separate source set for it, IMO no need to create a sub-project

Reply:
Done. In which I learned more about Gradle than ever before, even though the final change now looks tiny. Good idea, though.